editor/plugins/tile_map_editor_plugin.cpp

```diff
@@ -1448,8 +1459,30 @@ bool TileMapEditor::forward_gui_input(const Ref<InputEvent> &p_event) {
 			CanvasItemEditor::get_singleton()->update_viewport();
 			return true;
 		}
-	}
+	} else if (k.is_valid()) {
+
+		if (tool == TOOL_NONE) {
+
+			if (k->get_scancode() == KEY_SHIFT && k->get_command()) {
 
+				tool = TOOL_PICKING;
+				_update_button_tool();
+			}
+		} else if (tool == TOOL_PICKING) {
+
+#ifdef APPLE_STYLE_KEYS
+			if (k->get_scancode() == KEY_META) {
+#else
+			if (k->get_scancode() == KEY_CONTROL) {
```

Comment:
These tests are usually done with `KEY_MASK_CMD` which binds to Meta on OSX and Ctrl otherwise.

Reply:
To compare with `KEY_MASK_CMD` I'll need to use `k->get_scancode_with_modifiers()`, but this will not work in that case because this is an event release and `k->get_control()` and `k->get_command()` will always return `false`.

But I think we can remove these `ifdef` below:
https://github.com/godotengine/godot/blob/554c0ea90b4bc3b94d2c9a357d6f9c1db93abfe7/editor/plugins/tile_map_editor_plugin.cpp#L952-L971
We only need `mb->get_command()`, right?